README.md

```diff
@@ -37,15 +37,58 @@ For detailed information see the list of [supported migration features for Oracl
 
 For older releases see [Release Notes](RELEASE_NOTES.md).
 
-### v3.4.3
-
-This is a bugfix release for [v3.4.0](RELEASE_NOTES.md).
+### v3.5.0-rc.2
+
+#### Features
+
+- Several improvements in the execution of a migration:
+    - Swap **execution order of *Integrity* and *Logic* stage**.
+      Having indices in the Logic stage makes it easier to test performance of views, stored procedures and triggers
+      <p align="center">
+          <img src="./docs/pictures/release-notes/v3.5.0/change-stage-execution-order.png"></img>
+      </p>
+    - Move **creation of [check constraints](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-CHECK-CONSTRAINTS) from Logic into Structure stage**.
+      Checking the data during the data bulk load is negligible compared to the time needed to re-read the table from disk in the Logic stage.
+    - As consequence **functions are created in the Structure stage**, before the tables, since check constraints may use them.
```

Comment:
```suggestion
    - As a consequence, **functions are created in the Structure stage** prior to tables, since they may be used by check constraints.
```